login-workflow/src/components/WorkflowCard/WorkflowCard.tsx

```diff
@@ -33,7 +33,7 @@ const makeStyles = ({
         tablet: {
             height: MAX_CARD_HEIGHT,
             width: MAX_CARD_WIDTH,
-            borderRadius: theme.roundness * 2,
+            borderRadius: theme.roundness * 10,
```

Comment:
I think we need to add 24px of border radius to the card. `theme.roundness` is 4px so it should be multiple of 6.
```suggestion
            borderRadius: theme.roundness * 6,
```